lib/plugin.js

```diff
@@ -1,15 +1,13 @@
-module.exports = function pluginMethod (plugin) {
-  var self = this
-
+module.exports = function pluginMethod (hoodie, options, plugin) {
   if (typeof plugin === 'function') {
-    plugin(self)
+    plugin(hoodie, options)
   }
 
   if (typeof plugin === 'object') {
     Object.keys(plugin).forEach(function (key) {
-      self[key] = plugin[key]
+      hoodie[key] = plugin[key]
```

Comment:
we don’t. If you need access to `options` you have to pass in a function, I think that’s fair